datamodel/src/main/java/com/hartwig/serve/datamodel/ClinicalTrial.java

```diff
@@ -32,5 +32,14 @@ public abstract class ClinicalTrial implements Intervention {
     public abstract Set<String> therapyNames();
 
     @Nullable
-    public abstract String genderCriterium();
-}
\ No newline at end of file
+    public abstract GenderCriterium genderCriterium();
+
+    @NotNull
+    public abstract Set<String> urls();
+
+    @NotNull
+    public abstract Set<Indication> indications();
+
+    @NotNull
+    public abstract Set<MolecularProfile> molecularProfiles();
```

Comment:
I would put these more up top, though not all the way. I would put at least `urls` at the bottom.

Another suggestion, though not super strong opinion, would `molecularCriteria` be a better term for the profiles? It matches nicely with `genderCriterium` and is more specific overall